app.js

```diff
@@ -12,10 +16,25 @@ const notFoundMiddleware = require('./middleware/not-found');
 const errorHandlerMiddleware = require('./middleware/error-handler');
 const authenticationMiddleware = require('./middleware/authentication');
 
+app.set('trust proxy', 1);
+app.use(
+  rateLimiter({
+    windowMs: 15 * 60 * 1000,
+    max: 100,
+  })
+);
+app.use(express.json());
+app.use(helmet());
+app.use(cors());
+app.use(xss());
+
 app.use(express.static('./public'));
 app.use(express.json());
 
 // ROUTES
+app.get('/', (req, res) => {
+  res.send('Dear Diary API');
+});
```

Comment:
I want to deploy it with this entry point and without to see how the static files in the public will served so we'll test out both.

Reply:
No problem. Go ahead